dnd-bot/dnd_bot/logic/prototype/multiverse.py

```diff
@@ -1,9 +1,13 @@
 import copy
+import numpy as np
 
 
 class Multiverse:
     """contains the list of all the games (objects of Game class) that are being played"""
     games = dict()
+    square_size = 50
+    view_range = 4
```

Comment:
może te wartości wywalić jako stałe do jsona?

Reply:
wydaje mi się, że jest to niepotrzebne

Reply:
te wartości są stałe dla wszystkich gier

Reply:
no to stałe pythonowe poza funkcją